.github/workflows/continous-integration.yml

```diff
@@ -285,6 +292,7 @@ jobs:
       env:
         COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         COVERALLS_SERVICE_NAME: github
+        COVERALLS_PARALLEL: true
```

Comment:
I think this is what caused the huge drops / spikes in the coverage reporting. I'm following the docs [here](https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html) which say:

> If you are running multiple jobs in parallel and want coveralls.io to merge those results, you should set COVERALLS_PARALLEL to true in your environment:
>
> `COVERALLS_PARALLEL=true coveralls`

Reply:
Nice catch